src/css3/walker.rs

```diff
@@ -0,0 +1,272 @@
+use crate::css3::node::{Node, NodeType};
```

Comment:
The walker is just an easy way of displaying the tree. It probably has no use outside debugging purposes.